centreon/tests/e2e/cypress/e2e/Resources-status/04-timezone.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Timezone in monitoring
+    As a Centreon user
+    I want to be able to use timezones across the platform
+    To manage and visualize the monitoring resources more accurately
+
+Background:
+    Given a user authenticated in a Centreon server
+    And the platform is configured with at least one resource
+
+Scenario: Configuring timezone in user's profile
+   When user cliks on Timezone field in his profile menu
+   And user selects a Timezone / Location
+   And user saves the form
+   Then timezone information are updated on the banner
```

Comment:
Let's remain consistent: user vs the user

Reply:
seems not resolved
ex : `user saves the form` vs `the user creates a downtime on a resource`

Reply:
@kduret what do you mean ?

Reply:
its fixed now